starlite/openapi/path_item.py

```diff
@@ -87,7 +111,7 @@ def create_path_item(
 
             tags, security = extract_layered_values(route_handler)
             operation = Operation(
-                operationId=route_handler.operation_id or handler_name,
+                operationId=operation_id,
```

Comment:
the above logic is ok - but, the standard in OA is to use camelCase rather than snake case. Can you switch to using this or do you think differently?

Reply:
Sounds good, updated